structuredheadings/plugin.js

```diff
@@ -301,22 +317,23 @@
       setupElements(editor);
       setupCommands(editor);
       setupStyles(editor);
+      self.currentScheme = editor.config.autonumberDefaultScheme;
 
       editor.on("dataReady", function () {
         self.currentScheme = self.detectScheme();
       });
 
       //Format Dropdown
       editor.ui.addRichCombo("NumFormats", {
-        label: "Formats",
-        title: "Numbering Formats",
+        label: "Formatting",
+        title: "Text Formatting",
```

Comment:
[this](https://trello.com/c/b8fGNlrA/37-editor-structured-headings-numbering-1-1-1-1-1-1-new-customers#comment-58ffb2786289aeba5a336e99) specifies "Numbering" for this menu. Where did the "Text Formatting" label idea come from?